src/components/_common/IconicButton/iconicButton.css

```diff
@@ -0,0 +1,12 @@
+.iconic-button {
+  background-color: var(--primary-element-color);
+  padding: 6px;
+  border-radius: 50%;
+  font-size: 28px;
+  line-height: 1;
+  color: var(--secondary-text-color);
+}
+
+.iconic-button:hover {
+  color: var(--primary-text-color);
+}
```

Comment:
this code is duplicated. see burgerButton.css

Reply:
(better to live this one, and delete for burger)

Reply:
resolved